includes/class-edd-cli.php

```diff
@@ -1821,21 +1821,38 @@ public function display_legacy_payment_data( $args ) {
 	public function recalculate_download_sales_earnings() {
 		global $wpdb;
 
-		$downloads = $wpdb->get_results(
-			"SELECT ID
+		$step        = 0;
+		$offset      = 0;
+		$number      = 1000;
+		$has_results = true;
+		$total       = $wpdb->get_var(
+			"SELECT COUNT(ID)
```

Comment:
```suggestion
			"SELECT COUNT(*)
```

Ref: https://aaronfrancis.com/2022/mysql-count-star-slow